app/lib/zeebe-api/get-system-certificates.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+/**
+ * This file contains code adapted from https://github.com/microsoft/vscode-proxy-agent
+ *
+ * MIT License
+ *
+ * Copyright (c) 2014 Nathan Rajlich &lt;nathan@tootallnate.net&gt;
+ * Copyright (c) 2015 Félicien François &lt;felicien@tweakstyle.com&gt;
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * 'Software'), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+const cp = require('child_process');
+const os = require('os');
+const fs = require('fs/promises');
+let _cachedCertificates;
+
+module.exports = getSystemCertificates;
+
+/**
+ * Get certificates from the system keychain.
+ *
+ * @returns {Promise<string[]>}
+ */
+async function getSystemCertificates() {
+  if (_cachedCertificates) {
+    return _cachedCertificates;
+  }
+
+  _cachedCertificates = await readCaCertificates() || [];
+
+  return _cachedCertificates;
+}
+
+async function readCaCertificates() {
+  if (process.platform === 'win32') {
+    return readWindowsCaCertificates();
+  }
+  if (process.platform === 'darwin') {
+    return readMacCaCertificates();
+  }
+  if (process.platform === 'linux') {
+    return readLinuxCaCertificates();
+  }
+}
+
+async function readWindowsCaCertificates() {
+  const winCA = require('vscode-windows-ca-certs');
```

Comment:
I don't see this import added. Did we miss it?

Reply:
Hmm it looks so! i will add it

Reply:
OK this is now added. I triggered another build which should be tested by @philippfromme or @Skaiir (Windows).

Reply:
Works ✔️